src/main/java/com/softserveinc/ita/rozetka/page_objects/OrderPage.java

```diff
@@ -0,0 +1,36 @@
+package com.softserveinc.ita.rozetka.page_objects;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public class OrderPage {
+
+    public OrderPage setSurname(String surname) {
+        $x("//div[@class='form__row js-surname']/input").setValue(surname);
+        return this;
+    }
+
+    public OrderPage setName(String name) {
+        $x("//div[@class='form__row js-name']/input").setValue(name);
+        return this;
+    }
+
+    public BasketPage startEditingProductsInBasket() {
```

Comment:
still invalid, what product will be edited if multiples products there?

Reply:
the task of this method is to start editing
It opens Basket page for editing.  Item(s) of product(s) is/are chosen by user in another method(s)

Reply:
use singular then -> startEditingProductInBasket()